plugin/bullets.vim

```diff
@@ -297,6 +297,55 @@ fun! s:match_bullet_list_item(input_text)
 endfun
 " -------------------------------------------------------  }}}
 
+" Selection management ----------------------------------- {{{
+
+" These functions help us maintain the cursor or selection across operation
+"
+" When getting selection we record
+" 1. The start and end line of the selection
+" 2. Thd the offset of the start and end column the line end
+"
+" When setting the selection we set the start and end at the same _offset_
+" From the new line start and end. As we manipulate line prefixes, the
+" offset from the end represents the correct new cursor position
+fun! s:get_selection(is_visual)
```

Comment:
We need to pass in is_visual because this state is lost by the time the function is triggered.